client/schema/v1/index.js

```diff
@@ -426,9 +427,10 @@ export default () => {
           ]
         },
         training: {
-          title: 'Training',
+          title: 'Training record',
```

Comment:
Is this going to change the label for the section? i.e these will change:
![Screenshot 2025-01-10 at 17 29 49](https://github.com/user-attachments/assets/9c9b9273-e8d0-4210-9491-b72fd0e6598d)
![Screenshot 2025-01-10 at 17 30 03](https://github.com/user-attachments/assets/de9e9d05-834d-40a2-86b7-b4e8e0532abd)

To match the react components I think we need to add a sub heading in the table renderer - see my comment there...